src/Plaza/getPlazas.js

```diff
@@ -0,0 +1,16 @@
+export default async function getPlaza() {
+    //const apiURL = 'https://park-inn-ispp-be.herokuapp.com/plazas/all';
+    return await fetch('http://localhost:8080/plazas/all', {
+        method: 'GET',
+        credentials:'same-origin',
+        headers: {
+            'Accept': 'application/json',
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/park-inn-ispp/park-inn-frontend/pullRequest?prid=9107115)
```suggestion
            "Accept": 'application/json',
```